api/files/serializers.py

```diff
@@ -394,6 +394,14 @@ class FileSerializer(BaseFileSerializer):
         min_version='2.0', max_version='2.7',
     )
     target = TargetField(link_type='related', meta={'type': 'get_target_type'})
+    current_user_has_viewed = ser.SerializerMethodField(help_text='Whether the current user has already viewed the file')
+
+    def get_current_user_has_viewed(self, obj):
+        version = obj.versions.order_by('created').last()
+        if version and not self.context['request'].user.is_anonymous:  # This is to ensure compatibility with tests, ugh.
```

Comment:
@brianjgeiger if the user is unauthenticated it will return False, is that okay with FE?

Reply:
It would probably make more sense to return True. In the False case, we change the UI to make it look special. In the True case, it would appear to be default.